src/mscorlib/System.Private.CoreLib.csproj

```diff
@@ -356,6 +356,7 @@
     <Compile Include="$(BclSourcesRoot)\Internal\Runtime\Augments\EnvironmentAugments.cs" />
     <Compile Include="$(BclSourcesRoot)\Internal\Runtime\Augments\RuntimeThread.cs" />
     <Compile Include="$(BclSourcesRoot)\Internal\Console.cs" />
+    <Compile Include="$(BclSourcesRoot)\Internal\Padding.cs" />
```

Comment:
I used the filename Padding.cs because I noticed corefx used that name for the equivalent file.